src/encoded/static/components/donor.js

```diff
@@ -320,13 +308,6 @@ const MouseDonor = (props) => {
                             </div>
                         : null}
 
-                        {tagBadges ?
-                            <div className="tag-badges" data-test="tags">
-                                <dt>Tags</dt>
-                                <dd>{tagBadges}</dd>
-                            </div>
-                        : null}
-
```

Comment:
It looks like you're taking the internal tags off of the mouse donor and fly worm donor. We don't want those any more?

Reply:
They don’t exist in the mouse and fly worm donor schemas (any more? Not sure if they used to) — only the human schema.